addons/context_actions/CfgContext.hpp

```diff
@@ -137,6 +137,31 @@ class EGVAR(context_menu,actions) {
             statement = QUOTE([ARR_2(_selectedObjects,'DOWN')] call FUNC(setStance));
         };
     };
+    class Captives {
+        displayName = "$STR_ACE_Captives_DisplayName";
+        condition = QUOTE( \
```

Comment:
I think this should just be `isClass (configFile >> 'CfgPatches' >> 'ace_captives')`.